src/components/common/UserDetails.tsx

```
@@ -93,124 +76,40 @@ const UserDetails: React.FC<UserDetailsProps> = ({ userData }) => {
+					</HStack>
+				))}
+				{/* Custom fields, 2 per row */}
+				{userData?.customFields && userData.customFields.length > 0 && (
+					<Box>
+						<VStack spacing={2} align="stretch">
+							{chunkArray(userData.customFields, 2).map((row) => (
+								<HStack
+									key={row.map((f) => f.label).join('_')}
+									spacing={4}
+								>
+									{row.map((field, idx) => (
```

ESLint

[error] 'idx' is defined but never used.

(@typescript-eslint/no-unused-vars)